packages/site/src/content/AnimatedPresence/AnimatedPresenceNotes.mdx

```diff
@@ -0,0 +1,40 @@
+## Configuration
```

Comment:
Do we want to include a comment in the original file(s) for these extractions letting others know that this has technically been duplicated in the short term?

I would hate for us to make a correction in the original docs somewhere and then lose it when we cut out that work later.

Reply:
Yes! Great idea! I will do that in my next set of components